relayer/config/config.go

```diff
@@ -256,6 +243,15 @@ func (c *Config) GetWarpQuorum(blockchainID ids.ID) (WarpQuorum, error) {
 	return WarpQuorum{}, errFailedToGetWarpQuorum
 }
 
+func (c *Config) GetWarpRequirePrimaryNetworkSigners(blockchainID ids.ID) (bool, error) {
+	for _, s := range c.DestinationBlockchains {
+		if blockchainID.String() == s.BlockchainID {
+			return s.warpRequirePrimaryNetworkSigners, nil
+		}
+	}
```

Comment:
For the number of subnets we have for now, doing a linear search is fine, but it does kind of beg the question if we should have the blockchains in a map keyed by the `BlockchainID`. If it's a big extra lift because that would be too hard to parse straight from the config, that's fine.

Reply:
Yeah it's definitely not worth changing IMO for a one-time config parsing.